src/test/java/com/google/sps/UserTest.java

```diff
@@ -39,6 +40,82 @@ public void userNotEquals() {
     User userA = new User("John");
     User userB = new User("Not John");
 
-    Assert.assertFalse(userA.equals(userB));
+    Assert.assertFalse((userA).equals(userB));
   }
+
+  @Test
+  public void setIDAndGetID(){
+      User userA = new User("Jeff");
+      long expected = 1112;
+      userA.setID(expected);
+      Long result = userA.getID();
+
+      Assert.assertEquals(Long.valueOf(expected), result);
+  }
+
+  @Test
+  public void setSchoolAndGetSchool(){
+      User userA = new User("Ben");
+      String expected = "Stanford University";
+      userA.setSchool(expected);
+      String result = userA.getSchool();
+
+      Assert.assertEquals(expected, result);
+  }
+   // test for profileType here
+  /*
```

Comment:
In general, commented out code should not be checked in because it can cause confusion about whether the code is needed. Please consider removing or uncommenting out the code.

Reply:
Ok, won't check that in next time. Done